code/src/pages/Home.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Provider } from "react-redux";
+import { TodoList } from "../components/TodoList.js";
+import { createStore, combineReducers, configureStore } from "@reduxjs/toolkit";
+import { todos } from "../reducers/todos.js";
+import { Header } from "components/Header"
+import { CustomCheckbox } from 'components/CustomCheckbox';
+
+// Tell redux about our reducers
+const reducer = combineReducers({ todos: todos.reducer });
+
+// // Retrieve the existing state from localstorage if it exists
+const persistedState = localStorage.getItem("reduxState")
+  ? JSON.parse(localStorage.getItem("reduxState"))
+  : {};
+
+// Create the store using our reducers and the retrieved state
+const store = createStore(
```

Comment:
Stick to `configureStore` instead of `createStore`, since we're using Redux Toolkit and not the whole Redux library.
[https://redux-toolkit.js.org/api/configureStore](https://redux-toolkit.js.org/api/configureStore)